src/devices/PiJuice/PiJuicePower.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Iot.Device.PiJuiceDevice.Models;
+
+namespace Iot.Device.PiJuiceDevice
+{
+    /// <summary>
+    /// PiJuicePower class to support status of the PiJuice
+    /// </summary>
+    public class PiJuicePower
+    {
+        private readonly PiJuice _piJuice;
+
+        /// <summary>
+        /// PiJuicePower constructor
+        /// </summary>
+        /// <param name="piJuice">The PiJuice class</param>
+        public PiJuicePower(PiJuice piJuice)
+        {
+            _piJuice = piJuice;
+        }
+
+        /// <summary>
+        /// Gets the delay before the PiJuice removes power to the GPIO pins
+        /// </summary>
+        /// <returns>The delay in seconds</returns>
+        public byte GetPowerOff()
```

Comment:
A property would work `byte PowerOff` or even better `TimeSpan PowerOff`
For all time related delay, we tend to use TimeSpan to be consistent